custom/icds/tasks.py

```diff
@@ -111,6 +111,8 @@ def run_weekly_indicators():
 
         for user_id in generate_user_ids_from_primary_location_ids(domain, get_awc_location_ids(domain)):
             language_code = get_language_code(user_id, telugu_user_ids, marathi_user_ids)
+            if language_code != TELUGU:
```

Comment:
Would it be possible to use the location instead of language code? Will all states have different languages?